src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToJudgeAddlnInfoHandler.java

```diff
@@ -49,10 +57,20 @@ private CallbackResponse submitClaim(CallbackParams callbackParams) {
         String userId = idamClient.getUserInfo(callbackParams.getParams().get(BEARER_TOKEN).toString()).getUid();
         CaseData.CaseDataBuilder caseDataBuilder = caseData.toBuilder();
         String role = DocUploadUtils.getUserRole(caseData, userId);
+        List<Element<Document>> tobeAdded = caseData.getGeneralAppAddlnInfoUpload();
```

Comment:
Suggestion: lines 60-63 can be replace with               
 List<Element<Document>> tobeAdded = Optional.ofNullable(caseData.getGeneralAppAddlnInfoUpload()).orElse(new ArrayList<>());